lldb/unittests/Process/elf-core/ThreadElfCoreTest.cpp

```diff
@@ -120,7 +121,10 @@ TEST_F(ElfCoreTest, PopulatePrpsInfoTest) {
   ASSERT_EQ(prpsinfo_opt->pr_state, 0);
   ASSERT_EQ(prpsinfo_opt->pr_sname, 'R');
   ASSERT_EQ(prpsinfo_opt->pr_zomb, 0);
-  ASSERT_EQ(prpsinfo_opt->pr_nice, 0);
+  int priority = getpriority(PRIO_PROCESS, getpid());
+  if (priority == -1)
```

Comment:
Are you assuming that the priority of the debugged process is the same as the test process, or is the debugged process in fact the same as the test process?

Reply:
In this test the process for which the prpsinfo object is being populated is the test process. The issue I saw in the build kites was that, rather than the test process being prioritized at the default for linux, it got 5. So the fix here is just to make sure that, whatever the priority of the test process, what is verified that the populated value in the prpsinfo object matches the priority that we know the process runs as.